crates/hir-def/src/nameres.rs

```diff
@@ -262,7 +262,7 @@ impl DefMap {
         // NB: we use `None` as block here, which would be wrong for implicit
         // modules declared by blocks with items. At the moment, we don't use
         // this visibility for anything outside IDE, so that's probably OK.
-        let visibility = Visibility::Module(ModuleId { krate, local_id, block: None });
```

Comment:
Shouldn't the root module have public visibility? Without this change we'll lose crate completions since they arent visible from modules from other crates right now

(Note to self: comment is now obsolete, need to remove that)

Reply:
Yeah, makes sense to me